test/slonik/integration.ts

```diff
@@ -997,6 +997,23 @@ test('throw error with notices', async (t) => {
       t.assert(error.notices.length = 5);
     }
   }
+  await pool.end();
+});
+
+test('error messages include original pg error', async (t) => {
+  const pool = createPool(t.context.dsn);
+  const createPerson123 = sql`insert into person(id) values (123)`;
+
+  await pool.query(createPerson123);
+
+  const error = await t.throwsAsync(async () => {
+    return pool.query(createPerson123);
+  });
+
+  t.regex(
+    error.message,
+    /^Query violates a unique integrity constraint: duplicate key value violates unique constraint "person_pkey"$/,
```

Comment:
Any thoughts on getting rid of the "Query violates a unique integrity constraint" part completely? We'd still get the type safety by having the `UniqueIntegrityConstraintViolationError ` class, and the other part conveys the same information, but a bit more helpfully/specifically.

Reply:
I am of the inverse opinion that we shouldn't include: `duplicate key value violates unique constraint "person_pkey"` part.

I am generally of opinion that error messages should be static, and that details such as "which constraint" belong as properties of the error.